kio-client-java-spring/src/test/java/org/zalando/stups/clients/kio/spring/RestTemplateKioOperationsTest.java

```diff
@@ -95,8 +94,8 @@ public void testGetRangeOfApplications() throws ParseException {
         final ZonedDateTime modifiedAfter = now().minusDays(2);
 
         mockServer.expect(requestTo(
-                format("%s/apps?modified_before=%s&modified_after=%s", //
-                        BASE_URL, toIsoString(modifiedBefore), toIsoString(modifiedAfter)))) //
+                UriComponentsBuilder.fromHttpUrl(format("%s/apps?modified_before=%s&modified_after=%s", //
```

Comment:
can you please implement the test differently than the production code? otherwise it shows nothing (which is a as bad as my version before)